tests/core/parameter_validation/test_parameter_validation.py

```diff
@@ -37,7 +37,12 @@ def check(file_name, keywords):
     ])
 def test_parsing_errors(test):
     with pytest.raises(ParameterParsingError):
-        check(*test)
+        check_fails_with_message(*test)
+
+
+def test_array_type():
+    path = os.path.join(BASE_DIR, 'array_type.yaml')
+    load_parameter_file(path, 'array_type')
```

Comment:
```suggestion
    assert load_parameter_file(path, 'array_type')
```

Reply:
It will anyway pass as it is a truthy value -otherwise it would fail. Just a proposition for reader's clarity.